verify/trusted_root.pem

```diff
@@ -13,5 +13,4 @@ ZWwuY29tL0ludGVsU0dYUm9vdENBLmRlcjAdBgNVHQ4EFgQUImUM1lqdNInzg7SV
 Ur9QGzknBqwwDgYDVR0PAQH/BAQDAgEGMBIGA1UdEwEB/wQIMAYBAf8CAQEwCgYI
 KoZIzj0EAwIDSQAwRgIhAOW/5QkR+S9CiSDcNoowLuPRLsWGf/Yi7GSX94BgwTwg
 AiEA4J0lrHoMs+Xo5o/sX6O9QWxHRAvZUGOdRQ7cvqRXaqI=
------END CERTIFICATE-----
-
+-----END CERTIFICATE-----
```

Comment:
Why drop the trailing newline?

Reply:
We expected that there would be no remaining bytes after decoding the PEM certificate. Removing the newline character resolved this issue. Therefore, the newline was removed.